Framework/Algorithms/src/ConvertUnits.cpp

```diff
@@ -333,7 +340,7 @@ ConvertUnits::convertQuickly(API::MatrixWorkspace_const_sptr inputWS,
 */
 bool ConvertUnits::getDetectorValues(
     const Kernel::Unit &outputUnit, const Geometry::IComponent &source,
-    const Geometry::IComponent &sample, double l1, int emode,
+    const Geometry::IComponent &sample, double &l1, int &emode,
```

Comment:
This was a hang over from a an earlier refactoring attempt where I was hoping to only have the algorithms differ by implementations of getDetectorValues.  However it turned out to be more complex than that.

I'll revert this change.